### PR TITLE
Link TFConnect news to threefold_io RSS 

### PR DESCRIPTION
**Pull request by AlaaElattar**
- https://github.com/threefoldtech/threefold_connect_news/issues/5

https://github.com/user-attachments/assets/79a58cfc-c101-46a9-a7e0-06bd066c6a9e

### CODE REVIEW

File: app/lib/apps/news/news_screen.dart [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                return Padding(
+                  padding:
+                      const EdgeInsets.symmetric(horizontal: 8, vertical: 4.0),
+                  child: Card(
+                    color: Theme.of(context).colorScheme.background,
+                    elevation: 4.0,
+                    shape: RoundedRectangleBorder(
+                        borderRadius: BorderRadius.circular(12)),
+                    child: Padding(
+                      padding: const EdgeInsets.all(8.0),
```
Comment (AhmedHanafy725) [inline app/lib/apps/news/news_screen.dart:124]:
```suggestion
                      padding: const EdgeInsets.all(16),
```

File: app/lib/apps/news/news_screen.dart [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Row(
+                            children: [
+                              Image.asset(
+                                'assets/tft_icon.png',
+                                height: 20,
+                                width: 20,
```
Comment (AhmedHanafy725) [inline app/lib/apps/news/news_screen.dart:133]: this icon needs to have color based on the theme mode.

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]
it might have double spinners if the user scrolled twice, let's block the second one

**Comment by AlaaElattar** [comment]
https://github.com/user-attachments/assets/b211ae15-d6d9-428b-9615-90c4acce0889

File: app/lib/apps/news/news_screen.dart [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+                    crossAxisAlignment: CrossAxisAlignment.center,
+                    children: [
+                      Padding(
+                        padding: EdgeInsets.only(
+                            bottom: 4,
+                            top: MediaQuery.of(context).size.height * 0.4),
+                        child: const CircularProgressIndicator(),
+                      ),
+                      Text(
+                        'Loading Articles ...',
```
Comment (AhmedHanafy725) [inline app/lib/apps/news/news_screen.dart:119]:
```suggestion
                        'Loading Articles...',
```

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**